src/views/MainPage.vue

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable vue/no-parsing-error */
+<template>
+  <v-container>
+      <div v-for="film in films" :key="film.id" class="card-film">
+      <div class="card-film__img">
+        <img :src="'https://image.tmdb.org/t/p/w500' + film.poster_path" alt="poster">
+      </div>
+      <div class="card-film__descr">
+        <div class="card-film__descr__name">
+          Name of film: {{ film.original_title }}
+        </div>
+        <div class="card-film__descr__overview">
+          Overview: {{ film.overview }}
+        </div>
+      </div>
+      <v-btn @click="getDetailsOfMovie(film.id)" rounded color="green">More</v-btn>
+    </div>
+  </v-container>
+</template>
+<script>
+
+export default {
+  name: 'MainPage',
+  created () {
+    // console.log('Route: ', this.$route)
+    // console.log('Store: ', this.$store)
+    this.getTopFilms()
+  },
+  destroyed () {
+    this.$store.commit('destroyTopFilms')
+  },
+  computed: {
+    films () {
+      return this.$store.state.topFilms
+    }
+  },
+  methods: {
+    getTopFilms () {
+      this.$store.dispatch('getBestFilms')
+    },
```

Comment:
names should be consistent!
TopFilms, BestFilms, use only one name pattern

Reply:
Done